gui/client/index.html

```diff
@@ -102,6 +102,9 @@ <h3 class="panel-title">Game Status
               No data available.
             </div>
           </div>
+
+          <!-- GitHub Button -->
+          <a href="https://ainize.ai"><img src="./assets/ainize.svg" style="margin-top: -11px; margin-left: 1px;"></a>
```

Comment:
I'm not sure if I want this button visible when running locally or hosted on other places. Can we make this conditional so that this only shows when accessed under the `ainize.ai` domain?

Reply:
> I'm not sure if I want this button visible when running locally or hosted on other places. Can we make this conditional so that this only shows when accessed under the `ainize.ai` domain?

Good Idea. I'll reflect this